.eslintrc.js

```diff
@@ -9,13 +9,13 @@ module.exports = {
         project: 'tsconfig.eslint.json'
     },
     rules: {
-        'no-shadow': 'off',
+        '@typescript-eslint/no-shadow': 'off',
         'no-restricted-imports': [
             'warn',
             {
                 name: 'sprotty-protocol',
                 message:
-                    "The sprotty-protocol default exports are customized and reexported by GLSP. Please import from '@eclipse-glsp/client' instead"
+                    "The sprotty-protocol default exports are customized and reexported by GLSP. Please import from '@eclipse-glsp/protocol' instead"
```

Comment:
Is that true? It seems to me that even with https://github.com/eclipse-glsp/glsp-client/pull/253 we still re-export the sprotty types from '@eclipse-glsp/client'?

Reply:
The `@eclipse-glsp/protocol` package has a partial reexport of the needed `sprotty-protocl` tpyes (https://github.com/eclipse-glsp/glsp-client/blob/d81a1ea5335410d2daeaa0b869ed7573eaff6471/packages/protocol/src/index.ts#L43). There should be no reason to ever directly use the `sprotty-protocol` package in a GLSP context.

Reply:
I understand that we don't want `sprotty-protocol`. I was just confused because in all other linting rules we suggest to use `@eclipse-glsp/client` instead but here it is `@eclipse-glsp/protocol` but it seems that there is a re-export in both, the client and in the protocol package and I the server should not have a dependency on the client ;-)